src/utils/gitHubUtils.ts

```diff
@@ -148,6 +150,22 @@ export async function tryGetRemote(): Promise<string | undefined> {
     return;
 }
 
+export async function tryGetBranch(): Promise<string | undefined> {
```

Comment:
For "try" functions, I prefer if the "try/catch" wraps like the entire function. For example, maybe `git(localProjectPath);` could throw an error 🤷‍♂️

Reply:
🧤